docs/feature_mouse_keys.md

```diff
@@ -191,6 +191,7 @@ Recommended settings in your keymap’s `config.h` file:
 |`MOUSEKEY_MAX_SPEED`        |32       |Maximum cursor speed at which acceleration stops           |
 |`MOUSEKEY_TIME_TO_MAX`      |32       |Number of frames until maximum cursor speed is reached     |
 |`MOUSEKEY_FRICTION`         |24       |How quickly the cursor stops after releasing a key         |
+|`MOUSEKEY_STOP_SPEED`       |1        |Lowest possible decelerating velocity before it snaps to 0 |
```

Comment:
My Japanese isn't particularly good. Is it alright that I only updated the English documentation?

Reply:
Yeah, that's fine.